Fullstack-Development/README.md

```diff
@@ -0,0 +1,90 @@
+# Frontend Roadmap
```

Comment:
Add linters to the entire file. Leave one blank line after every header

Reply:
I have made the remaining changes. Can you please give a bit more details on the `linter` part? Should I use this [linter](https://github.com/igorshubovych/markdownlint-cli#readme)?

Reply:
Also could you please give some details on how to add it?

Reply:
Linters are just how your document is formatted. You added spaces between all Headers, that is linting

Reply:
> Linters are just how your document is formatted. You added spaces between all Headers, that is linting
Okay. Thanks for sharing :)
Are the changes fine now?

Reply:
> Linters are just how your document is formatted. You added spaces between all Headers, that is linting

Okay. Thanks for sharing :)
Are the changes fine now?